src/components/timeline/Tooltip.svelte

```diff
@@ -80,6 +87,18 @@
   function tooltipText(): string {
     let tooltipText = '';
 
+    if (gaps.length) {
+      // For now we render a single static "No Value" message for any number of gaps,
```

Comment:
Was there any discussion around labeling which resource the "no value" is attributed to?

Reply:
Yeah that's a good question. I don't think we've given this too much thought yet. Previously it just used the same tooltip as the resource values, including an ID and start time, but we decided to drop that because it's not super useful information. I think we might just need to see how this is used in practice for a bit and see if we do need more descriptive labels.